Example/MyPlayground.playground/Contents.swift

```diff
@@ -0,0 +1,39 @@
+//: Playground - noun: a place where people can play
+
+import UIKit
+import PlaygroundSupport
+import Sourcing
+
+struct Train {
+    var id: String
+    var title: String
+}
+
+class TrainCell: UITableViewCell, ConfigurableCell {
+    typealias DataSource = Train
+    func configure(with train: DataSource) {
+        self.textLabel?.text = train.title
+    }
+}
+
+class TableViewController: UITableViewController {
+    let trainCellConfiguration = CellConfiguration<TrainCell>(cellIdentifier: "TrainCell")
+    var dataProvider: ArrayDataProvider<Train>!
+    var dataSource: TableViewDataSource<Train>!
+    var trains: Array<Train> = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(TrainCell.self, forCellReuseIdentifier: "TrainCell")
+        for i in 0..<20 {
```

Comment:
Creating some example trains make the sourcing example harder to unterstand.
This playground should be about Sourcing and not looping to create object. By moving in to train you hide the complexity of creating the example objects